packages/fuels-contract/src/contract.rs

```diff
@@ -859,7 +864,7 @@ impl MultiContractCallHandler {
     }
 
     /// Create a [`FuelCallResponse`] from call receipts
-    pub fn get_response<D: Tokenizable + Debug>(
+    pub async fn get_response<D: Tokenizable + Debug>(
```

Comment:
it feels weird to me that now this function is an `async`. Isn't there a way to get the `tx_id` without having to do an async call? I feel like it's an extra network and I'm wondering if it's necessary to do it.

Reply:
@Br1ght0ne Thoughts on this? It seems like it's the only unresolved conversation here. We should ship this soon! 😁

Reply:
ddf32b3 instead gets `tx_id` from `call/simulate` calls, building on the assumption that you definitely should have `tx_id` if you own `receipts` (to be passed to `get_response`).